### PR TITLE
Use more precise datetime formatting in job status

### PR DESCRIPTION
**Pull request by romasku**
Closes https://github.com/neuro-inc/platform-client-python/issues/2043

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-PYTEST_ARGS=
+PYTEST_ARGS= --rich-gen
```
Comment (asvetlov) [inline Makefile:3]: Why the flag is always on? OMHO it can modify generated files during the test run on CI without letting us know.

**Review by asvetlov — CHANGES_REQUESTED** [review]
Add changelog please

File: Makefile (hunk 1 above)
Comment (romasku) [reply Makefile]: I just forgot to remove it, sorry.

**Review by asvetlov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**